metr_docs/README.md

```diff
@@ -168,10 +168,13 @@ With that done, install the rest of the Python dependencies:
 (redashvenv1) $ pip3 install wheel  # "wheel" needs to be installed by itself first
 (redashvenv1) $ pip3 install --upgrade black ruff launchpadlib pip setuptools
 (redashvenv1) $ pip3 install poetry
+(redashvenv1) $ poetry install --with dev
```

Comment:
The `dev` is also being installed in the line bellow. The problem is that it also installs all_ds, which is only necessary in the production environment. Those are the dependencies for the data sources.

I think we can change the readme to have something like
```
(redashvenv1) $ poetry install --with dev
# If you need to have the data sources dependencies locally, also install them with
(redashvenv1) $ poetry install --only all_ds
```

Reply:
thanks a lot , yeah i tested it and then changed it in the readme :D